CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,83 @@
+# Code of conduct for `govcookiecutter`
+
+Contributors to this repository hosted on `ukgovdatascience` are expected to follow the Contributor Covenant Code of
+Conduct, and those working within Her Majesty's Government are also expected to follow the Civil Service Code.
+
+## Civil Service Code
+
+The Civil Service Code can be found [here][civil-service-code].
+
+## Contributor Covenant Code of Conduct
+
+### Definitions
+
+Where this Code of Conduct says:
+
+- "Project", we mean this `govcookiecutter` GitHub repository;
+- "Maintainer", we mean the `ukgovdatascience` organisation owners; and
+- "Leadership", we mean both `ukgovdatascience` organisation owners, line managers, and other leadership within GDS.
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make
+participation in our project and our community a harassment-free experience for everyone, regardless of age, body size,
+disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education,
+socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+### Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
```

Comment:
Are we using British English or American English? If the former, it is spelt below:
```suggestion
Examples of unacceptable behaviour by participants include:
```